inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -744,6 +744,7 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         LayerParams attrs = {node->get_friendly_name(), "GRUSequence",
                              details::convertPrecision(node->get_output_element_type(0))};
         auto res = std::make_shared<RNNSequenceLayer>(attrs);
+        IE_ASSERT(res != nullptr);
```

Comment:
I propose to remove these asserts. As I know `make_shared` throws a `bad_alloc` if we have any issues with allocation.

Reply:
fixed